code/datums/diseases/transformation.dm

```diff
@@ -123,14 +122,12 @@
 					"<span class='warning'>You have a craving for bananas.</span>", "<span class='warning'>Your mind feels clouded.</span>")
 	stage5 = list("<span class='warning'>You feel like monkeying around.</span>")
 
-/datum/disease/transformation/jungle_fever/do_disease_transformation(mob/living/carbon/affected_mob)
-	if(affected_mob.mind && !IS_INFECTED_MONKEY(affected_mob.mind))
-		affected_mob.mind.add_antag_datum(/datum/antagonist/monkey)
+/datum/disease/transformation/jungle_flu/do_disease_transformation(mob/living/carbon/affected_mob)
+	if(affected_mob.mind)
 		affected_mob.monkeyize()
 		ADD_TRAIT(affected_mob, TRAIT_VENTCRAWLER_ALWAYS, type)
```

Comment:
should this be kept? It was supposed to be given along with the antag datum since "monkey fever" monkeys are "smarter" or something

Reply:
fair enough

Reply:
does this still need that mind check then? it just turns you into a monkey, doesnt give you antag anymore

Reply:
very guh